consensus/leader.go

```diff
@@ -311,6 +311,7 @@ func (consensus *Consensus) onCommit(msg *msg_pb.Message) {
 		return
 	}
 
+	// TODO(audit): verify signature on hash+blockNum+viewID (add a hard fork)
```

Comment:
why is it a hard fork though, this is just at consensus level, i guess then it needs to be driven down to the blockchain level too

Reply:
Because the signed data changed, then the verification and all need to be changed too.